libmate-desktop/mate-colorsel.c

```diff
@@ -1226,7 +1230,7 @@ override_background_color (GtkWidget *widget,
 
   provider = gtk_css_provider_new ();
 
-  css = g_strdup_printf ("* { background-color: %s;}",
+  css = g_strdup_printf ("* { color: %s;}",
```

Comment:
ah, maybe coupled with that it works…  but why override with CSS then manually draw it?  it'd be simpler just to paint a given color, wouldn't it?  And if it's more complex, this will change behavior.